examples/eg2_properties.cpp

```diff
@@ -22,20 +22,29 @@ void eg2_properties()
     std::cout << "Width: " + std::to_string(resolutions[0].first) + ", Height: " + std::to_string(resolutions[0].second) << std::endl;
 
 
+
     // Open the first camera in the biggest resolution
     std::cout << "Open the first camera..." << std::endl;
     camera.open(cameraDeivceList[0], 
         resolutions[resolutions.size() - 1].first,
         resolutions[resolutions.size() - 1].second
     );
-
+    std::cout << "show DirectShow properties" << std::string(*camera.getDirectShowProperties()) << std::endl;
```

Comment:
I want to keep the console clear. It could be great if you can remove it.

Reply:
ok,got it